draft-ietf-httpbis-header-structure.md

```diff
@@ -100,22 +100,20 @@ shown here.
 
 This document uses the Augmented Backus-Naur Form (ABNF) notation of {{!RFC5234}}, including the DIGIT, ALPHA and DQUOTE rules from that document. It also includes the OWS rule from {{!RFC7230}}.
 
-This document uses algorithms to specify normative parsing behaviours, and ABNF to illustrate the on-wire format expected. Implementations MUST follow the normative algorithms, but MAY vary in implementation so as the behaviours are indistinguishable from specified behaviour. If there is disagreement between the algorithms and ABNF, the specified algorithms take precedence.
+This document uses algorithms to specify normative parsing/serialisation behaviours, and ABNF to illustrate expected syntax. Implementations MUST follow the normative algorithms, but MAY vary in implementation so as the behaviours are indistinguishable from specified behaviour. If there is disagreement between the algorithms and ABNF, the specified algorithms take precedence.
```

Comment:
serialization*

Or does the IETF not use en-US?

Reply:
I let the RFC editor figure it out.

Reply:
Section 3.1 of RFC 7322 says:

> The RFC publication language is English.  Spelling may be either American or British, as long as an individual document is internally consistent.  Where both American and British English spelling are used within a document or cluster of documents, the text will be modified to be consistent with American English spelling.

Reply:
Wow, I wonder how that hasn't been a problem in practice. Maybe because the IETF doesn't really do direct cross-references to algorithms and such?